.github/workflows/deploy-workflow.yml

```diff
@@ -165,3 +165,23 @@ jobs:
           --set global.sql.datasource.host=${POSTGRES_HOST} \
           --set global.sql.datasource.hostForpostgresqlClient=${POSTGRES_CLIENT_HOST} \
           --set global.sql.datasource.url=${POSTGRES_URL}
+
+      - name: install datahub-monitoring helm
+        shell: bash
+        env:
+          KUBE_NAMESPACE: ${{ secrets.kube_namespace }}
+          DASHBOARD_NAME: datahub-status-dashboard
+        run: |
+          envsubst < helm_deploy/monitoring/datahub-networkpolicy.yaml \
+          | kubectl apply -f - --namespace=${KUBE_NAMESPACE}
+
+          kubectl create configmap ${DASHBOARD_NAME} \
+          --from-file=helm_deploy/monitoring/datahub-dashboard.json \
+          --dry-run \
+          --output yaml \
+          | kubectl label -f- \
+          --dry-run \
+          --output yaml \
+          --local grafana_dashboard=${DASHBOARD_NAME} \
+          | kubectl apply -f- \
```

Comment:
This looks good but I don't fully understand the end to end deployment. How does this config get picked up by cloud platform grafana?

Reply:
We deploy a `configMap` to the namespace that contains the dashboard from the `-dashboard.json`. If it [meets certain criteria, it'll get picked up](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/prometheus.html#save-the-dashboard-json-in-a-configmap) by Cloud Platform's grafana deployment crawler.

Reply:
Ah nice - the crawler was the bit I was missing